nsot/conf/settings.py

```diff
@@ -425,7 +426,35 @@
 IP_VERSIONS = ('4', '6')
 
 # Whether to compress IPv6 for display purposes, for example:
-# - Exploded (default): 2620:0100:6000:0000:0000:0000:0000:0000/40 
+# - Exploded (default): 2620:0100:6000:0000:0000:0000:0000:0000/40
 # - Compressed: 2620:100:6000::/40
 # Default: True
 NSOT_COMPRESS_IPV6 = True
+
+#########
+# Hooks #
+#########
+# from nsot.util.core import hook_delivery
+
+# Map actual model action names to cosmetic event names
+# E.g: resource.added => will trigger for created action
+HOOK_ACTIONS = {
```

Comment:
I'd like to see all of these new settings prefixed with `NSOT_`. I actually want to sweep the old settings to prefix them as well at some point.

(Obviously `HOOK_EVENTS` is exempt since it's required by `django-rest-hooks`.)